bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1478,17 +1478,18 @@ namespace JsRTApiTest
     {
         JsValueRef result = JS_INVALID_REFERENCE, callbackState = JS_INVALID_REFERENCE, exception = JS_INVALID_REFERENCE;
         JsValueType cbStateType = JsUndefined;
+        const wchar_t *script = _u("new Promise((res, rej) => res()).then(() => 1)");
```

Comment:
I agree- dealing with making NativeTests xplat is a separate issue, and is already tracked with #2590. I'll poke that issue again. For now, you can leave it as wchar_t, we can clean this up later.